dlrm_s_pytorch.py

```diff
@@ -142,7 +142,13 @@ def create_mlp(self, ln, sigmoid_layer):
 
     def create_emb(self, m, ln):
         emb_l = nn.ModuleList()
+        # save the numpy random state
+        np_rand_state = np.random.get_state()
         for i in range(0, ln.size):
+            if ext_dist.my_size > 1:
+                if not i in self.local_emb_indices: continue
+            # Use per table random seed for Embedding initialization
+            np.random.seed(self.l_emb_seeds[i])
```

Comment:
We should probably also do this in the general case as well to keep things consistent (cc: @mnaumovfb) ?

Reply:
Can you add  a "WARNING: ..." comment explaining that the random seed is initialized per table in parallel mode (with MPI backend), while they are initialized once in the sequential run? I think for now this will be sufficient.

Reply:
The initialization (and per table random seeds) is consistent across sequential and parallel mode within this dist_exp branch. It only differs compared to master branch. So, this warning can be misleading...

Reply:
Ok, understood. Thank you for clarifying. In that case let's leave it as is for now.